content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1311,6 +1326,8 @@ Defines a multi-field ID (composite ID) on the model.
 | Name     | Required | Type               | Description                                                      |
 | -------- | -------- | ------------------ | ---------------------------------------------------------------- |
 | `fields` | **Yes**  | `FieldReference[]` | A list of field names - for example, `["firstname", "lastname"]` |
+| `name`   | **No**   | `String`           |                                                                  |
```

Comment:
These could use descriptions: 
name: The name the Client Api will expose for the id argument covering all fields i.e. `fullName: { firstName: Paul, lastName: Panther}`
map: The name of the underlying Primary Key constraint in the database.